eva-server/src/test/java/uk/ac/ebi/eva/server/ws/RegionWSServerIntegrationTest.java

```diff
@@ -139,4 +140,16 @@ public void testVepVersionAndVepCacheVersionFilter() {
         }
     }
 
+    @Test
+    public void testIllegalLimitParameter() {
+        String testRegion = "20:60000-80000";
+        String annotationVepVersion = "78";
+        String annotationVepCacheversion = "78";
+        int limit = 100000;
+        String url = "/v1/segments/" + testRegion +
```

Comment:
I know we have been doing it this way (concatenation the string) but maybe if we use a builder it would be easier to read. Something like:
```
        String url = UriComponentsBuilder.fromUriString("")
                                         .path("/v1/segments/")
                                         .path(testRegion)
                                         .path("/variants")
                                         .queryParam("species", "mmusculus_grcm38")
                                         .queryParam("annot-vep-version", annotationVepVersion)
                                         .queryParam("annot-vep-cache-version", annotationVepCacheversion)
                                         .queryParam("limit", limit)
                                         .build().toString();
```